src/Datadog.Trace.ClrProfiler.Managed.Core/Datadog.Trace.ClrProfiler.Managed.Core.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
+
+    <!-- NuGet -->
+    <Version>1.14.0</Version>
```

Comment:
Should we keep the assembly version semi-constant (`1.0.0`) like we do for `Datadog.Trace.AspNet`?

Reply:
Good idea, I'll change that

Reply:
Fixed in d3c5fd865a5332058a9ff2f50ce2242164354550